code/game/gamemodes/blob/blob_report.dm

```diff
@@ -10,6 +10,8 @@
 				if(bomb && bomb.r_code)
 					if(is_station_level(bomb.z))
 						bomb.r_code = nukecode
+						message_admins("The send_intercept proc in the blob gamemode has triggered, the station has been sent a message requesting that the station is destructed along with the nuke codes, [nukecode].")
+						log_admin("The send_intercept proc in the blob gamemode has triggered, the station has been sent a message requesting that the station is destructed along with the nuke codes, [nukecode].")
```

Comment:
Since these are not debug logs, they should not be using proc names and references to the code. Instead they should be explicitly geared towards administration purposes, i.e. an explanation of what's going on in-game terms that any admin can understand without having to have any understanding of code.

Reply:
Implemented